src/fu-util.c

```diff
@@ -1251,12 +1260,17 @@ fu_util_refresh (FuUtilPrivate *priv, gchar **values, GError **error)
 	}
 
 	/* open file */
-	return fwupd_client_update_metadata (priv->client,
-					     values[2],
-					     values[0],
-					     values[1],
-					     NULL,
-					     error);
+	if (!fwupd_client_update_metadata (priv->client,
+					   values[2],
+					   values[0],
+					   values[1],
+					   NULL,
+					   error))
+		return FALSE;
+
+	/* TRANSLATORS: success message -- the user can do this by-hand too */
+	g_print ("%s\n", _("Successfully refreshed metadata manually"));
```

Comment:
I don't think you need the word manually, especially considering this is also called from the optional systemd unit

Reply:
Something that might be interesting is to mention "$x firmware now available", or something similar.

Reply:
I don't think the client knows this kind of data, although I concede the number of different devices might be interesting to show. A daemon uint32 property perhaps?

Reply:
Yeah, that sounds good

Reply:
Something like `UniqueDeviceCount` or `UniqueComponentIdCount` `SupportedComponentCount`? The alternative is we just say how many *remotes* we refreshed rather that how many devices are supported now which would be much easier, but a bit less cool.

Reply:
Another way that might be easy and still cool is to have the client count up how many devices have the `supported` flag and it can go to:
`Successfully refreshed metadata, 15 local devices now supported`

Reply:
I've added `Successfully downloaded new metadata: 3 local devices supported`